.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+    // Use IntelliSense para saber los atributos posibles.
+    // Mantenga el puntero para ver las descripciones de los existentes atributos.
+    // Para más información, visite: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Archivo actual",
+            "type": "python",
+            "request": "launch",
+            "program": "${file}",
+            "console": "integratedTerminal",
+            "justMyCode": false
+        }
+    ]
+}
```

Comment:
(Not the author here but a past contributor)

I'm not familiar with this launch.json file, was it added by mistake ?

Reply:
yeah, my bad that was added by mistake